app/controllers/recipes_controller.rb

```diff
@@ -9,10 +9,14 @@ def new
     @recipe = Recipe.new
   end
 
+  def show
+    @recipe = Recipe.find(params[:id])
+  end
+
   def create
```

Comment:
Maybe we could handle `RecipeGeneratorServiceError` in this method. Rescue any exception coming from Service in case of error and show an alert, what do you think?